### PR TITLE
Translate the developer guide #181

### PR DESCRIPTION
**Pull request by lixiaojiee**
## What is the purpose of the change

translate the new contribute guide

## Brief changelog

please see the picture below:

wx20181122-211445 2x

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo-website/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `docsite start`, and make sure it works as expected.
- [ ] Make sure no files under build directory is added.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by lovepoem** [comment]
"新贡献者向导"，从 https://github.com/apache/incubator-dubbo-website/pull/188/files 看起来你没有提交上

File: docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+
+邮件列表是 Dubbo 官方推荐的讨论基本上所有与 Dubbo 相关内容的方式，请点击 [issue](https://github.com/apache/incubator-dubbo/issues/1393) 了解更多关于邮件列表订阅的内容
+
+* dev@dubbo.incubator.apache.org：开发邮件列表，您在使用或者开发 Dubbo 的过程中遇到的任何问题，都可以在这里进行提问。
+* commits@dubbo.incubator.apache.org：所有的提交内容都会推送到这个邮件列表，如果您对 Dubbo 的进展感兴趣，可以订阅这个邮件列表。
+* issues@dubbo.incubator.apache.org：所有的 JIRA [issues](https://issues.apache.org/jira/projects/DUBBO/issues) 和修改信息都会推送到这个邮件列表。Dubbo 社区已经决定使用 github issues 代替 JIRA issues，因此大部分 issues 将由 github issues 进行跟踪。JIRA issues 用于跟踪 ASF 相关问题。
+
+### 报告问题
+
+### 发送提交请求
```
Comment (lovepoem) [inline docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md]: 建议“pull request” 不要翻译成“提交请求”，保持原状，有两三处

Comment (lixiaojiee) [reply docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md]: 好，我修改一下

File: docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md [hunk 2]
```diff
@@ -1 +1,31 @@
 # 新贡献者向导
+
+这篇向导旨在给正在准备贡献 Dubbo 的新手提供指导。
```
Comment (lovepoem) [inline docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md]: “贡献 Dubbo 的新手” 修改为“向Dubbo提交贡献的新手”

Comment (lixiaojiee) [reply docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md]: 好，我改下

File: docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+
+### 邮件列表描述
+
+邮件列表是 Dubbo 官方推荐的讨论基本上所有与 Dubbo 相关内容的方式，请点击 [issue](https://github.com/apache/incubator-dubbo/issues/1393) 了解更多关于邮件列表订阅的内容
```
Comment (lovepoem) [inline docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md]: “邮件列表是 Dubbo 官方推荐的讨论基本上所有与 Dubbo 相关内容的方式”   修改为   “邮件列表是Dubbo官方推荐的讨论方式，所有与Dubbo相关的内容都可以在这里讨论”

Comment (lixiaojiee) [reply docs/zh-cn/developers/contributor-guide/new-contributor-guide_dev.md]: 已经重新提交，请查看

**Comment by lovepoem** [comment]
Since you have completed the translation of this part, please leave a message after https://github.com/apache/incubator-dubbo-website/issues/181

**Comment by lovepoem** [comment]
LGTM

**Comment by lixiaojiee** [comment]
thank you

**Pull request merged**